packages/dtslint-runner/src/main.ts

```diff
@@ -161,16 +161,16 @@ export async function runDTSLint({
   return allFailures.length;
 }
 
-function getExpectedFailures(onlyRunAffectedPackages: boolean) {
+function getExpectedFailures(onlyRunAffectedPackages: boolean, dependents: readonly string[]) {
   if (onlyRunAffectedPackages) {
     return undefined;
   }
```

Comment:
These lines were supposed to be removed. Idk why I missed that in the commit, but I'll update this.